processing/ip_handler_test.go

```diff
@@ -136,6 +138,16 @@ func TestIPHandler(t *testing.T) {
 		t.Fatalf("expected %d forwarded BLF alerts, seen less (%d)", 2,
 			len(fwhandler.Entries))
 	}
+
+	var i interface{}
```

Comment:
Nitpicking, and +20 years of coding, but `i` is like bound to be an integer in any language ;)
Maybe call this the `result`? (I know why `i` is used, like `s` is for string..) Like I said, nitpicking!

Reply:
Good point.